packages/11ty/_includes/components/figure/image/image-tag.js

```diff
@@ -0,0 +1,26 @@
+const { html } = require('~lib/common-tags')
+const path = require('path')
+
+/**
+ * Image Tag for figures that are static images
+ * @param      {Object} eleventyConfig  eleventy configuration
+ *
+ * @param      {Object} params `figure` data from `figures.yaml`
+ * @property   {String} alt The alt text for the image
+ * @property   {String} src The src path for the image
+ * @return     {String}  An <img> element
+ */
+module.exports = function(eleventyConfig) {
+  const { imageDir } = eleventyConfig.globalData.config.params
+
+  return function ({
+    alt='',
+    src=''
+  }) {
```

Comment:
With only two destructured variables it may be easier to read on one line?
```suggestion
  return function ({ alt='', src='' }) {
```